tools/DevicesApiTester/Infrastructure/HexStringUtilities.cs

```diff
@@ -22,13 +22,13 @@ public static string FormatByteData(byte[] data, int perGroup = 4, int perLine =
             }
 
             int dataLength = data.Length;
-            int lineCount = dataLength / perLine;
+            int lineCount = (int)Math.Ceiling((double)dataLength / perLine);

```

Comment:
>Ceiling [](start = 38, length = 7)

why Ceiling and not Floor?

Reply:
I actually didn't try Floor, but the thinking was Ceiling would round up to get the next lineCount where Floor would round down and cut off the line including bytes that didn't fill the entire line.

Would you like for me to experiment to make sure my logic is off?

Reply:
If you are tested it and ensured it is working as expected when we round up, then that is fine. if you didn't test this particular scenario, then it would be better to try it,

Reply:
thanks @shaggygi for confirminh